docs/howto/mirroring.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Mirroring
+parent: How-To
+description: Mirroring allows replicating commits between lakeFS installations in different geo-locations/regions
+---
+
+# Mirroring
```

Comment:
This should have BETA written all over it :)

Reply:
Added in the title!